atd-etl/app/process_cris_cr3.py

```diff
@@ -103,4 +103,4 @@ def wait(int):
 end = time.time()
 hours, rem = divmod(end-start, 3600)
 minutes, seconds = divmod(rem, 60)
-print("Finished in: {:0>2}:{:0>2}:{:05.2f}".format(int(hours),int(minutes),seconds))
\ No newline at end of file
+print("Finished in: {:0>2}:{:0>2}:{:05.2f}".format(int(hours),int(minutes),seconds))
```

Comment:
Add newline at EOF

This is an unrelated change that I'm tag-along'ing into this PR.